torch_np/_funcs_impl.py

```diff
@@ -730,50 +709,29 @@ def triu_indices(n, k=0, m=None):
 def tril_indices_from(arr: ArrayLike, k=0):
     if arr.ndim != 2:
         raise ValueError("input array must be 2-d")
-    result = torch.tril_indices(arr.shape[0], arr.shape[1], offset=k)
-    return tuple(result)
+    # Return a tensor rather than a tuple to avoid a graphbreak
+    return torch.tril_indices(arr.shape[0], arr.shape[1], offset=k)
 
 
 def triu_indices_from(arr: ArrayLike, k=0):
     if arr.ndim != 2:
         raise ValueError("input array must be 2-d")
-    result = torch.triu_indices(arr.shape[0], arr.shape[1], offset=k)
-    # unpack: numpy returns a 2-tuple of index arrays; torch returns a 2-row tensor
-    return tuple(result)
+    # Return a tensor rather than a tuple to avoid a graphbreak
+    return torch.triu_indices(arr.shape[0], arr.shape[1], offset=k)
 
 
 def tri(N, M=None, k=0, dtype: DTypeLike = None, *, like: NotImplementedType = None):
     if M is None:
         M = N
     tensor = torch.ones((N, M), dtype=dtype)
-    tensor = torch.tril(tensor, diagonal=k)
-    return tensor
+    return torch.tril(tensor, diagonal=k)
 
 
-# ### nanfunctions ###  # FIXME: this is a stub
+# ### nanfunctions ###
 
 
-def nanmean(
-    a: ArrayLike,
-    axis=None,
-    dtype: DTypeLike = None,
-    out: Optional[OutArray] = None,
-    keepdims=None,
-    *,
-    where: NotImplementedType = None,
-):
-    # XXX: this needs to be rewritten
-    if dtype is None:
-        dtype = a.dtype
-    if axis is None:
-        result = a.nanmean(dtype=dtype)
-        if keepdims:
-            result = torch.full(a.shape, result, dtype=result.dtype)
-    else:
-        result = a.nanmean(dtype=dtype, dim=axis, keepdim=bool(keepdims))
-    if out is not None:
-        out.copy_(result)
-    return result
+def nanmean():
+    raise NotImplementedError
```

Comment:
Yeah, these all should go. This weirdness was to be able to port nanXXX tests, so should probbaly be redone in one go. This PR or a separate one, whichever works.